src/components/Sum.jsx

```diff
@@ -1 +1,27 @@
-// Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Sum.scss';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
```suggestion
export const Sum = ({ a,  b }) => (
```
No need to split it to multiple lines if there are less than 3 arguments